app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -42,6 +43,14 @@ import org.mozilla.fenix.mvi.getAutoDisposeObservable
 import org.mozilla.fenix.mvi.getManagedEmitter
 import kotlin.math.roundToInt
 
+fun SessionBundleStorage.archive(sessionManager: SessionManager) {
+        save(sessionManager.createSnapshot())
+        sessionManager.sessions.filter { !it.private }.forEach {
```

Comment:
Do we allow archiving private sessions? I'll assume not for the moment.

Reply:
@colintheshots Negative. The session storage filters out all private sessions before archiving.

I figured for now we would keep your private sessions around if there is one in memory.